test/resources/checkers/javascript-syntax-error.js

```diff
@@ -1,3 +1,3 @@
-/** A missing semicolon */
+/** A bad if */
 
-var foo = "Hello world);
+if ( /* nothing here */ ) // comment
```

Comment:
I didn't want the error to be on the last character of the line (or beyond?)... Not sure if it mattered, but it was easier to verify this way.